### PR TITLE
PT-621 | Limit file upload size to under 2MB

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

File: graphene_linked_events/schema.py [hunk 1]
```diff
@@ -715,15 +719,23 @@ class ImageMutationResponse(ObjectType):
     result_text = String()
 
 
+def _validate_image_upload(image):
+    if image.size > settings.MAX_UPLOAD_SIZE:
+        raise UploadImageSizeExceededError(
+            f"Upload file size cannot greater than {settings.MAX_UPLOAD_SIZE} bytes"
```
Comment (tuomas777) [inline graphene_linked_events/schema.py:725]: This sentence seems to be missing a verb.

File: graphene_linked_events/schema.py [hunk 2]
```diff
@@ -715,15 +719,23 @@ class ImageMutationResponse(ObjectType):
...
+
+
 class UploadImageMutation(Mutation):
     class Arguments:
         image = UploadImageMutationInput()
 
     response = Field(ImageMutationResponse)
 
-    @staff_member_required
+    # @staff_member_required
```
Comment (tuomas777) [inline graphene_linked_events/schema.py:735]: Hmm, intentional?

Comment (quyenlq) [reply graphene_linked_events/schema.py:735]: ugh.. forgot to ctrl + Z it after testing 😅

Comment (tuomas777) [reply graphene_linked_events/schema.py]: That would have been my first guess 🙂

**Review by tuomas777 — APPROVED** [review]
This looks good to me.

**Pull request merged**